packages/coinstac-ui/test/e2e/test.js

```diff
@@ -0,0 +1,140 @@
+const Application = require('spectron').Application;
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const path = require('path');
+
+const electronPath = path.join(__dirname, '../..', 'node_modules', '.bin', 'electron');
+const appPath = path.join(__dirname, '../..');
+const mocksPath = path.join(__dirname, 'mocks.js');
+
+const EXIST_TIMEOUT = 4000;
+const USER_ID = 'test';
+const PASS = 'test';
+const CONS_NAME = 'e2e-consortium';
+const CONS_DESC = 'e2e-description';
+const PROJECT_NAME = 'e2e-files';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const app = new Application({
+  path: electronPath,
+  env: { NODE_ENV: 'test' },
+  args: [appPath, '-r', mocksPath],
+});
+
+describe('Testing::e2e', () => {
+  before(() => {
+    return app.start();
+  });
+
+  after((done) => {
+    app.stop().then(() => { done(); });
```

Comment:
[`spectron.Application#stop` returns a `Promise`](https://github.com/electron/spectron#stop), so you don’t need to use a callback here. From [spectron’s example usage](https://github.com/electron/spectron#using) they check `spectron.Application#isRunning`, too. You can write this as:

```js
after(() => {
  if (app && app.isRunning()) {
    return app.stop();
  }
});
```